### PR TITLE
Revamp of Clinical Reminders, version 2

### PR DESCRIPTION
**Pull request by bradymiller**
Ongoing project (based on bootstrap-updates code). Continued from following PRs:
1. https://github.com/openemr/openemr/pull/2872
2. tywrenn#11
3. bradymiller#12
4. Next step is this PR!

### CODE REVIEW

**Comment by sjpadgett** [comment]
@bradymiller I could help ya out later today if you assign me something....

**Comment by bradymiller** [comment]
hi @sjpadgett , Thanks for the offer. I'll do a quick review/sweep/fix on this code and then I'll have a better idea where things stand on this (maybe it will be ready at that point; fingers crossed :crossed_fingers: ).

File: interface/super/rules/controllers/edit/template/criteria_target.php [hunk 1]
```diff
@@ -0,0 +1,106 @@
+<?php
+    /**
```
Comment (bradymiller) [inline interface/super/rules/controllers/edit/template/criteria_target.php]: PLACEHOLDER

File: interface/super/rules/controllers/edit/view/add_criteria.php [hunk 2]
```diff
@@ -1,40 +1,56 @@
...
-  <tr>
-        <td class="title"><?php echo xlt('Rule Edit'); ?></td>
-        <td>
-            <a href="index.php?action=detail!view&id=<?php echo attr_url($ruleId); ?>" class="iframe_medium btn btn-secondary" onclick="top.restoreSession()">
-                <span><?php echo xlt('Cancel'); ?></span>
-            </a>
-        </td>
-  </tr>
-</table>
+<?php
```
Comment (bradymiller) [inline interface/super/rules/controllers/edit/view/add_criteria.php]: PLACEHOLDER

**Comment by bradymiller** [comment]
first review iteration done :)
(still gotta do more detailed review/testing to get an idea of how much work will be needed on this)

**Comment by tywrenn** [comment]
@bradymiller What are we doing with this now that it's merged?

**Comment by bradymiller** [comment]
It's going to be migrated into yet another PR (number 5) based on the master branch :)
(likely do in next couple days)

**Comment by bradymiller** [comment]
Migrated this PR to main repo:
https://github.com/openemr/openemr/pull/2928

**Pull request closed without merging**